spec/chromium-spec.js

```diff
@@ -86,20 +86,6 @@ describe('chromium feature', () => {
     })
   })
 
-  describe('sending request of http protocol urls', () => {
```

Comment:
why delete this test ?

Reply:
This test is essentially sending CORS requests in `file://` and the request always fails, the test passes by accident. I found it when testing http protocol code.

The thing it wants to test (that sending http request should not crash) is already covered by other tests, we don't need to keep this invalid test.